xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+     to allow downloading all charts present in a chart repository in bulk and
+     moreover to extract all container image URLs from the charts.
+     <link xlink:href="https://github.com/containers/skopeo">skopeo</link> is
+     used to download all the images referred to in the Helm charts from their
+     respective registry.
+    </para>
+    <para>
+     Helm charts will be provided to the internal network by a webserver and
+     refer to the container images hosted on the internal registry mirror.
+    </para>
+    <para>
+     Once mirroring is configured, you will not have to modify Dockerfile(s) or
+     &kube; manifests to use the mirrors. The requests are passed through
+     the container engine which forwards them to the configured mirrors. For
+     example: All images with a prefix <literal>registry.suse.com/</literal>
+     will be automatically pulled from the configured (internal) mirror instead.
+    </para>
+    <para>
+     For further information on registry mirror configuration, refer to
+     <xref linkend="sec.admin.velum.registry" /> and
+     <xref linkend="sec.admin.velum.mirror" />.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.requirements">
+   <title>Requirements</title>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.machines">
+    <title>Mirror Servers</title>
+    <note>
+     <title>Shared Mirror Server</title>
+     <para>
+      If you have multiple &productname; clusters or a very large amount of
+      nodes accessing the mirrors, you should increase the sizing of CPU/RAM.
+     </para>
+     <para>
+      Storage sizing depends on your intended update frequency and data
+      retention model. If you want to keep snapshots or images of repository
+      states at various points, you must increase storage size accordingly.
+     </para>
+    </note>
+    <para>
+     You will need to provide and maintain at least two machines in addition to
+     your &productname; cluster. These mirror servers will reside on the
+     external part of your network and the internal (airgapped) network
+     respectively.
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>External</term>
+      <listitem>
+       <para>
+        This machine will host the <literal>&rmtool;</literal> for RPM packages
+        and the <literal>container image registry</literal> for container
+        images.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
```

Comment:
Here we suggest 500G of storage but later on why recommend at least 128G for the external storage. My understanding is that the external storage is suppose to transfer all the useful data from the external mirror to the internal one so if >=128G is all we need, then perhaps we can tune down the disk reqs on the mirror as well.

Reply:
Again, I don't really know. If each OS repository has like 30-40GB of data and you download a bunch of container images, you will run out of space quickly. For example you run this operation and keep the results for the mirror in a specific folder for rollback. You have this data 2-3-5-7 etc. times. So you quickly run into hundreds of Gigs of storage needed but the amount you transfer to the internal mirror is much smaller.

That's my perspective though, we haven't done any benchmarking of this use case to tell us how much data actually gets downloaded/created from this.

Reply:
OK no problem. My question was mostly because of the difference between 128G and 500G in the recommendations.